### PR TITLE
Add health endpoint

### PR DESCRIPTION
**Pull request by gb-beng**
Possibly a bit too much for such a simple endpoint, but trying to generalize and adopt existing patterns did help me a lot to get acquainted with it. Let me know if you have any questions!

~Before merging the relevant config should be present. See https://github.com/beeldengeluid/x-omgeving/pull/601 for that.~ (done)

Closes https://github.com/beeldengeluid/beng-lod-server/issues/190.

Testing checklist:
- Copy the config added to `settings-example.py` to your config.
- Start the server
- Check the response at `/ping` `/pong` and most imporantly `/health`.

### CODE REVIEW

File: src/apis/health/health_api.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+api = Namespace(
+    "Health",
+    description="Overall health of the API.",
+)
+
+
+@api.hide
+@api.route("health")
+class Health(Resource):
+    def get(this):
```
Comment (wmelder) [inline src/apis/health/health_api.py:18]: should be: `def get(self):`

**Comment by wmelder** [comment]
@gb-beng It is working, but why is there only one dependency?

File: src/apis/health/health_api.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+        logging.info("Determining health of dependencies")
+        health_timeout_sec = current_app.config["HEALTH_TIMEOUT_SEC"]
+        dependencies = [
+            Dependency(
+                "SPARQL_ENDPOINT",
+                current_app.config["SPARQL_ENDPOINT_HEALTH_URL"],
+            )
+        ]
+        dependency_health: Tuple[Dependency, DependencyHealth] = [
```
[the remark this answers is not in the document]
Comment (gb-beng) [reply src/apis/health/health_api.py:27]: Great will fix! Didn't have `mypy` yet in this branch...

**Review by wmelder — APPROVED** [review]
Running the server gives me proper results, so I think this PR is OK!

**Comment by jblom** [comment]
@gb-beng @wmelder tested it as well and works fine. Nicely done with unit tests , the use of dataclasses, and no more endpoints in the server.py :-)

**Comment by wmelder** [comment]
@jblom Now the config: https://github.com/beeldengeluid/x-omgeving/pull/601

**Pull request merged**